### PR TITLE
Cleans up some awful code from the ticker

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
This PR cleans up two awful pieces of code from the ticker. A 2 **hour** long `spawn()` and a `while(1)` infinite loop. These have been replaced with much better solutions.

## Why It's Good For The Game
Code should not be shitcode

## Changelog
:cl: AffectedArc07
tweak: Optimised part of the Ticker Subsystem code
/:cl:

### CODE REVIEW

File: code/controllers/subsystem/statistics.dm [hunk 1]
```diff
@@ -0,0 +1,28 @@
+SUBSYSTEM_DEF(statistics)
+	name = "Statistics"
+	wait = 6000 // 10 minute delay between fires
+	runlevels = RUNLEVEL_GAME | RUNLEVEL_POSTGAME // Only count time actually ingame to avoid logging pre-round dips
+	offline_implications = "Player count and admin count statistics will no longer be logged to the database. No immediate action is needed."
```
Comment (AffectedArc07) [inline code/controllers/subsystem/statistics.dm:5]: This was made into a subsystem for the sake of putting the procs in one place, and incase we ever decide to start logging more statistics in the future (Gives a good branching point). 

That or ill just kill this when I do `SSblackbox`

**Comment by Spacemanspark** [comment]
Ah yes pulling out ticks from the code I see

**Comment by AffectedArc07** [comment]
Spark, youre really starting to **tick** me off here

**Comment by Spacemanspark** [comment]
Just one more pun to tick off the checklist

**Pull request merged**